setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -49,35 +39,35 @@ EKS_CLUSTER_CA_DATA=$(aws cloudformation describe-stacks --stack-name spinnaker-
 SPINNAKER_INSTANCE_PROFILE_ARN=$(aws cloudformation describe-stacks --stack-name spinnaker-managing-infrastructure-setup --query 'Stacks[0].Outputs[?OutputKey==`SpinnakerInstanceProfileArn`].OutputValue' --output text)
 ```
 
+## Set up the managed account
 
-## Managed Account - Base setup
-
-In each of managed account, create role that can be assumed by Spinnaker
+In each of managed accounts, create a IAM role that can be assumed by Spinnaker:
 
 > This needs to be executed in managing account as well.
 
 ```bash
-curl -O https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/managed.yaml  
+curl -O https://www.spinnaker.io/setup/install/providers/aws/managed.yaml  
 
 aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --template-file managed.yaml \
 --parameter-overrides AuthArn=$AUTH_ARN ManagingAccountId=$MANAGING_ACCOUNT_ID --capabilities CAPABILITY_NAMED_IAM
 ```
 
-## kubectl configurations
+## `kubectl` and `heptio authenticator` configurations
 
-Before you proceed next, ensure to configure [kubectl and heptio authenticator for aws is installed and configured](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html)
+1. Configure [kubectl and heptio authenticator for aws is installed and configured](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html) on the workstation/instance where you are running Halyard from. Alternatively, if you are using Halyard container, you must be on version >=1.5.0.
 
-Also, when an Amazon EKS cluster is created, the IAM entity (user or role) that creates the cluster is added to the Kubernetes RBAC authorization table as the administrator. Initially, only that IAM user can make calls to the Kubernetes API server using kubectl.
+ Also, when an Amazon EKS cluster is created, the IAM entity (user or role) that creates the cluster is added to the Kubernetes RBAC authorization table as the administrator. Initially, only that IAM user can make calls to the Kubernetes API server using `kubectl`.
```

Comment:
You'll need to indent each of these four spaces to get them to line up as list-item paragraphs, and to keep numbering from resetting to 1.